modules/keyframes.js

```diff
@@ -3,13 +3,14 @@
 var prefix = require('./prefix');
 
 var kebabCase = require('lodash/string/kebabCase');
+var isNode = require('detect-node');
```

Comment:
Instead of `detect-node`, we should use a copy of React's execution environment (since we shouldn't be deep linking into React): https://github.com/facebook/react/blob/2aeb8a2a6beb00617a4217f7f8284924fa2ad819/src/vendor/core/ExecutionEnvironment.js